### ISSUE TITLE
KEYS 

### ISSUE THREAD

**Issue by KAMPER225** [SunwellWoW/Sunwell-TBC-Bugtracker#4048]
Description: 

Hello! Why can not I buy the key to the heroic level of complexity, having a HONORED reputation with a faction, for example Thrallmar ??? I can buy recipes and everything else that requires exactly this reputation, but the key is not. Please correct this BUG. The key says that I must have a HONORED reputation for its acquisition. And so everywhere and everywhere it works correctly, on all servers.
How it works: 
having a REVERED reputation

How it should work: 
having a HONORED reputation

Source (you should point out proofs of your report, please give us some source):

**Comment by Arom** [comment]
At this stage of the game the reputation required for the key is Revered. Problem is with your client having data from another realm, clear Cache and eventually the WTF folder, then it should display correctly.

**Comment by kyotpls** [comment]
This is wrong. However it's a visual bag stating that you need honored to buy the key, when in fact you need **revered**

**Comment by firngrod** [comment]
The keys were indeed changed from revered to honored in patch 2.3, as can be seen in the notes from this article on wow.gamepedia.com as of July 2008:
https://wow.gamepedia.com/index.php?title=Reservoir_Key&oldid=1513801

The question is then wether this server decides to emulate the progression in instances of retail, releasing instances in tiers and adjusting this along the same timeline, or whether they go by the game patch.  I honestly don't know, but I have noticed bear mounts from ZA if I'm not mistaken.  They were from the same patch (2.3.0)

EDIT:  I looked it up, the mounts I have seen are from the store, not from ZA, which is scheduled for release in November.  The question is then whether the reputation levels are kept at revered until that time as well.
